bundles/org.openhab.binding.homematic/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -127,6 +127,21 @@
 				<default>2048</default>
 				<advanced>true</advanced>
 			</parameter>
+			<parameter name="useAuthentication" type="boolean">
+				<label>Use authentication to access the gateway</label>
```

Comment:
Labels should be: <25 chars, max 2-3 words and properly capitalized.
(after this is fixed, don't forget top run the mvn i18n plugin)

Reply:
Done